packages/cms-lib/api/__mocks__/hubdb.js

```diff
@@ -0,0 +1,538 @@
+const hubdb = jest.genMockFromModule('../hubdb');
```

Comment:
I think for api helpers that don't have side effects we shouldn't need to create our own mocks and can instead leverage jest mocks on their own together with `mockReturnValue` and other friends for working mocked functions. Make sense?

Reply:
I think so. I updated this, curious if that's along the lines of what you were thinking?

Reply:
Looks good. It also is fine to keep test fixtures/data in json or js files separately. Doing so can be helpful, if the same test data is useful across multiple tests and test files.

Reply:
Good point. I added these as separate files. Not sure if that's preferred vs keeping everything in one file (seen it done both ways).